### PR TITLE
Replace text field

### LINKED ISSUE #2007: Wrong color of "password" on E2E
This must be slipped through.
@AndyScherzinger aka master of UI, can you have a look there? 👍

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix #2007 

 

Signed-off-by: tobiaskaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/main/res/layout/setup_encryption_dialog.xml [hunk 1]
```diff
@@ -29,32 +28,26 @@
         android:id="@+id/encryption_status"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_margin="10dp"/>
+        android:layout_margin="10dp" />
```
Comment (AndyScherzinger) [inline src/main/res/layout/setup_encryption_dialog.xml]: can you raise this to 16dp (```standard_margin```)?

**Comment by AndyScherzinger** [comment]
👍 just a minor comment

**Comment by AndyScherzinger** [comment]
Nice, cc @mario for review :)

**Comment by AndyScherzinger** [comment]
@mario last one for today - I promise :D

**Comment by mario** [comment]
Can we please use Butterknife for all the views? Thanks!

**Comment by tobiasKaminsky** [comment]
We can, for new screens or as an enhancement for next milestone(s).
But this should be a quick fix and I do not want to change too much here.

**Comment by mario** [comment]
👍

**Pull request merged**